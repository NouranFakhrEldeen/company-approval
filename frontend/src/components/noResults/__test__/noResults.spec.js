import React from 'react';
import { cleanup } from '@testing-library/react';
// import { HashRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NoResults from '../noResults';


describe('testing no Results', ()=>{
  afterEach(cleanup);


  //   snapshot
  test('should do snapshot', () => {
    const input = renderer.create(
      <NoResults
      />
    ).toJSON();
    expect(input).toMatchSnapshot();
  });

});