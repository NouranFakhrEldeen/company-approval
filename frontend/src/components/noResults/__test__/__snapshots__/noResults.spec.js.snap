// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing no Results should do snapshot 1`] = `
<div
  data-testid="noResults"
>
  <h4
    className="text-center no-results"
  />
</div>
`;
